context-free.md

```diff
@@ -2,21 +2,27 @@
 
 ##  Who is this programming language for?
 
+This programming language appears to be for anyone interested in vector-based digital art that does not want to go through the hassle of learning a more complete graphics editor, like Photoshop. Although the language itself is not extremely complicated, it seems geared toward someone who has some programming experience.
 
 ## What is easy to do in this language? Why is it easy?
 
```

Comment:
I think identifying the "terminal shapes" as we called them in class is important. It's also fairly easy to implement randomization.